### DISCUSSION TITLE
Key features for new game developers to use AIR

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#2021] [General]
As the whole internet is discussing migration from Unity, I thought, can it be used to increase AIR userbase. What do you think, @ajwfrost ?

What features of AIR can be used to market it among newbie game developers? Here's what I've thought of:
- Possibility to publish iOS game from a Windows computers
- Low system requirements - easily runs on a 5-year-old laptop
- Possibility to biuld games for Android, iOS, Windows, MacOS and Linux from the same codebase
- Object-oriented language Action Script 3 with well established structures and patterns
- No "percentage fees" - free tier for earnings below $50K or fixed pricing if above it
- Starling, powerful graphic engine

**Comment by ghost** [comment]
@GeneralVimes Sorry, but where did you find those discussing about Unity?

**Reply by GeneralVimes** [reply]
Oh, it a big buzz these days: due to the statement of Unity CEO, due to merging of Unity and Ironsource, and due to inability of Unity inhouse gamedev team to complete a game using Unity :)

Even a Unity3D reddit added a link to a Godot reddit into the top.
